filters/resize.go

```diff
@@ -53,6 +77,16 @@ func (r *resize) CreateFilter(args []interface{}) (filters.Filter, error) {
 		return nil, err
 	}
 
+	if len(args) == 3 {
+		f.proportion, err = parse.EskipBoolArg(args[2])
+
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		f.proportion = true
```

Comment:
`keepAspectRatio` should be false by default, as it's true for the `longerEdgeResize` filter